tests/unit/spire_test.go

```diff
@@ -71,4 +71,36 @@ spire-server:
 			Expect(notes).Should(ContainSubstring("join_token"))
 		})
 	})
-})
+	Describe("spire-server.keyManager.aws_kms", func() {
+		It("plugin set ok", func() {
+			objs, err := ValueStringRender(chart, `
+spire-server:
+  keyManager:
+    awsKMS:
+      enabled: true
+      region: us-west-2
+      plugin_data: {}
+    disk:
+      enabled: false
+`)
+			Expect(err).Should(Succeed())
+			notes := objs["spire/charts/spire-server/templates/configmap.yaml"]
+			Expect(notes).Should(ContainSubstring("\"aws_kms\": {"))
+        })
```

Comment:
The spacing seems off in this file. Can you try running `go fmt -w tests/unit/spire-test.go` on it?

Reply:
done